cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Actions.hs

```diff
@@ -502,3 +513,22 @@ runGovernanceActionHardforkInitCmd
     firstExceptT GovernanceActionsCmdWriteFileError . newExceptT $
       conwayEraOnwardsConstraints eon $
         writeFileTextEnvelope outFile (Just "Hardfork initiation proposal") proposalProcedure
+
+carryHashChecks
```

Comment:
Can you add haddock?

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/915/commits/7cd0239376a393072691418ebf942ded280ed57c